src/fetch-data.js

```diff
@@ -0,0 +1,63 @@
+import { renderData, changeBackground } from './render-data';
+
+const flash = document.getElementById('flash'); // for showing errors
+
+async function fetchWeather(location) {
+  const api = '1627b8ae6ced3746531173abad9b4d06';
+  const unit = 'metric';
```

Comment:
- [ ] You need to add an option to either make the calls on Fahrenheit or Celsius depending on the user's preferences or build a toggle button to switch between the two of them after getting the result ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png)).